### ISSUE TITLE
New Fork

### ISSUE THREAD

**Issue by phal0r** [fechanique/cordova-plugin-fcm#543]
Anyone interested to collaborate on a new fork and release a new version of this?

**Comment by boltex** [comment]
Yes - i would like to add a simple feature (on android) where if the app is kept alive with a wakelock (when minimized) that the incoming notifications would still call the cordova app's javascript FCMPlugin.onNotification function. 

I'm not well versed enough in java/android to know rapidly what to change to get this to work. If you can give me a hint or suggestion it would be greatly appreciated. (I would also like to fork and contribute to this additional feature)

thank you & good day to you!
